subsys/bluetooth/mesh/rpr_cli.c

```diff
@@ -358,7 +365,7 @@ static void link_timeout(struct k_work *work)
 	struct bt_mesh_rpr_cli *cli = CONTAINER_OF(k_work_delayable_from_work(work),
 						   struct bt_mesh_rpr_cli, link.timeout);
 
-	if (cli->link.state != BT_MESH_RPR_LINK_IDLE) {
+	if (bearer.link != BEARER_LINK_IDLE) {
```

Comment:
Why switched to bearer?

Reply:
Tried to explain in the comment above: https://github.com/zephyrproject-rtos/zephyr/pull/55782/files#r1136912613